.github/workflows/project.yml

```diff
@@ -13,32 +13,40 @@ concurrency:
 
 jobs:
   integration:
+    strategy:
+      matrix:
+        version: [14, 16, 18]
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: ./.github/actions/node/setup
       - run: yarn install
-      - uses: ./.github/actions/node/14
-      - run: yarn test:integration
-      - uses: ./.github/actions/node/16
-      - run: yarn test:integration
-      - uses: ./.github/actions/node/18
```

Comment:
there's no way to use the matrix approach with our composite actions (at least that I can think of). Our composite actions do not bring much in this case anyway IMO